spec/services/pii/cacher_spec.rb

```diff
@@ -2,45 +2,47 @@
 
 describe Pii::Cacher do
   let(:password) { 'salty peanuts are best' }
+  let(:hashed_password) { user.unlock_password(password) }
   let(:user) { create(:user, password: password) }
   let(:profile) { build(:profile, :active, :verified, user: user, pii: { ssn: '1234' }) }
-  let(:diff_user) { create(:user, password: password) }
-  let(:diff_profile) { build(:profile, :active, :verified, user: diff_user, pii: { ssn: '5678' }) }
+  let(:diff_profile) { build(:profile, :verified, user: user, pii: { ssn: '5678' }) }
   let(:user_session) { {} }
 
   subject { described_class.new(user, user_session) }
 
   describe '#save' do
     before do
+      allow(FeatureManagement).to receive(:use_kms?).and_return(false)
       profile.save!
     end
 
-    it 'writes re-encrypted PII to user_session' do
-      server_encrypted_pii = subject.save(password)
+    it 'writes decrypted PII to user_session' do
+      server_encrypted_pii = subject.save(hashed_password)
 
       expect(server_encrypted_pii).to be_a String
-      expect(server_encrypted_pii).to_not match '1234'
-      expect(user_session[:encrypted_pii]).to eq server_encrypted_pii
+      expect(server_encrypted_pii).to match '1234'
+      expect(user_session[:decrypted_pii]).to eq server_encrypted_pii
     end
 
-    it 'allows specific profile to be re-encrypted' do
+    it 'allows specific profile to be decrypted' do
       diff_profile.save!
-      server_encrypted_pii = subject.save(password, diff_profile)
+      server_encrypted_pii = subject.save(hashed_password, diff_profile)
 
       expect(server_encrypted_pii).to be_a String
       expect(server_encrypted_pii).to_not match '1234'
-      expect(server_encrypted_pii).to_not match '5678'
-      expect(user_session[:encrypted_pii]).to eq server_encrypted_pii
+      expect(server_encrypted_pii).to match '5678'
+      expect(user_session[:decrypted_pii]).to eq server_encrypted_pii
```

Comment:
These expectations are confusing to me. I would expect the session variable name to correspond to what it is being compared to. Expecting decrypted PII to match Server-Encrypted PII is not intuitive. Could one or the other be renamed?